.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow for github actions
```

Comment:
Remove all the comments and please follow the format of https://github.com/sindresorhus/awesome/pull/1794/files

Reply:
@sindresorhus, I removed the comments and added fetch depth. I'm not if you wanted anything else under the "follow the format" category but I: 1. changed the job from `run_awesome_lint` to `lint` and removed the top level "name". 

If there is anything else you want changed feel free to change yourself or tag me when you respond and I'll fix asap.